aries-site/src/pages/templates/reverse-navigation.mdx

```diff
@@ -1,3 +1,36 @@
-import { ComingSoon } from '../../components';
+import { Image } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
+import {
+  NameValueListFontWeightSecondPreview,
+} from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
```

Comment:
For consistency, we haven't been using ":" in headings.

```suggestion
## When to Use
```

or we could do something like:
```suggestion
## When to Use

Reverse navigation should be applied when:
```